README.md

```diff
@@ -220,7 +230,7 @@ options:
         "git_push": {
             "changed": true,
             "error": "",
-            "output": "remote: Resolving deltas:   0% (0/1)        \rremote: Resolving deltas: 100% (1/1)        \rremote: Resolving deltas: 100% (1/1), completed with 1 local object.        \nTo https://github.com/studyly/git-acp-test.git\n   3ba9041..4596d9d  master -> master\n"
+            "output": "remote: Resolving deltas:   0% (0/1)        \rremote: Resolving deltas: 100% (1/1)        \rremote: Resolving deltas: 100% (1/1), completed with 1 local object.        \nTo https://github.com/lvrfrc87/git-acp-test.git\n   3ba9041..4596d9d  master -> master\n"
```

Comment:
Renamed by mistake?

Reply:
No this was intentional as it referes to the original test repo